parlai/utils/testing.py

```diff
@@ -314,13 +314,15 @@ def distributed_train_model(opt: Opt) -> Tuple[Dict[str, Any], Dict[str, Any]]:
         if 'dict_file' not in opt:
             opt['dict_file'] = os.path.join(tmpdir, 'model.dict')
 
-        parser = mp_train.setup_args()
-        popt = _forced_parse(parser, opt)
-
         # we need a prebuilt dictionary
         parser = build_dict.setup_args()
+        popt = _forced_parse(parser, opt)
```

Comment:
...what... what is going on over here?

Reply:
forced parse probably isn't necessary but i just need to build the dict file before calling distributed train